baus/datasources.py

```diff
@@ -21,22 +21,25 @@
 # define new settings files- these have been subdivided from the
 # general settings file
 # this is similar to the code for settings in urbansim_defaults
-@orca.injectable('hazards', cache=True)
-def hazards():
-    with open(os.path.join(misc.configs_dir(), "hazards.yaml")) as f:
+@orca.injectable('run_setup', cache=True)
+def run_setup():
+    with open("../run_setup.yaml") as f:
         return yaml.load(f)
 
 
-@orca.injectable('policy', cache=True)
-def policy():
-    with open(os.path.join(misc.configs_dir(), "policy.yaml")) as f:
-        return yaml.load(f)
+@orca.injectable('inputs_dir', cache=True)
+def inputs_dir(run_setup):
+    return run_setup['inputs_dir']
+
 
+@orca.injectable('outputs_dir', cache=True)
+def outputs_dir(run_setup):
+    return run_setup['outputs_dir']
 
-@orca.injectable('inputs', cache=True)
-def inputs():
 
-    with open(os.path.join(misc.configs_dir(), "inputs.yaml")) as f:
+@orca.injectable('policy', cache=True)
+def policy():
+    with open(os.path.join(orca.get_injectable("inputs_dir"), "policy.yaml")) as f:
```

Comment:
`run_setup['inputs_dir']` is globally available since `run_setup` is stored with orca, but I think the benefit of this setup is that the code establishes a place in which to find the inputs path, so that if the inputs path location were to change it gets changed in one place, rather than throughout the code.

Reply:
Resolving this one but please re-open as needed!